src/agoradatatools/etl/load.py

```diff
@@ -160,3 +160,21 @@ def dict_to_json(df: dict, staging_path: str, filename: str) -> str:
     json.dump(df_as_dict, temp_json, cls=NumpyEncoder, indent=2)
     temp_json.close()
     return temp_json.name
+
+
+def list_to_json(df: list, staging_path: str, filename: str) -> str:
+    """Converts a list into a JSON file.
+
+    Args:
+        df (list): List to be converted to a JSON file
+        staging_path (str): Path to staging directory
+        filename (str): name of JSON file to be created
+
+    Returns:
+        str: Returns a string containing the name of the new JSON file
+    """
+
+    temp_json = open(os.path.join(staging_path, filename), "w+")
+    json.dump(df, temp_json, cls=NumpyEncoder, indent=2)
+    temp_json.close()
+    return temp_json.name
```

Comment:
Generally, a context managed open is preferred like:

```
    with open(os.path.join(staging_path, filename), "w+") as temp_json:
          json.dump(df, temp_json, cls=NumpyEncoder, indent=2)
          return temp_json.name
```

This is so you don't need to be concerned about calling `.close()`, which is a valid way of accomplishing this, however, if this is the approach you want to take the `.close()` should be within a `finally` block so it's guaranteed to execute.

Reply:
Hmm, I do like this approach better than what I was doing. I was trying to copy what the other functions are doing. Feedback please: Should I...
1. Update just this one function with the preferred context managed open
2. Update all of the X to json functions with the preferred context managed open
3. Leave things as they are and create a Jira ticket for updating the functions to use the preferred context managed open

Thoughs?

Reply:
I would:

1. Update any of the code you are already touching to following this approach
2. Log a tech debt ticket to go back and look at the other areas of the code

Generally, the mantra I follow is: "Leave the code in a better place than when I started". That needs to be balanced with the scope of the change, the time you have to make the changes, and the time it's going to take to validate the change. Some minor things are probably not worth fixing if it means there is a significant effort required to test the change.

Reply:
I agree, update your own code and make a ticket for anything else you notice. I'm not sure who to assign the issue to so it doesn't get lost in the ether, maybe Jess?

Reply:
Thank you both for the feedback! I'll update the function I wrote, create a tech debt Jira ticket and assign it to Jess :)

Reply:
@JessterB I need help figuring out where to create this Jira ticket 🙃

Reply:
I'll jump in: Go to JIRA (https://sagebionetworks.jira.com/), click on "Create" in the top bar, for project select "Model AD Explorer (MG)", Issue type = "Task" (Jess will change it if she wants something different), assign it to Jess, don't worry about filling in Team or Validator.

Reply:
Thank you @jaclynbeck-sage, I missed this while I was out last week. What Jaclyn said works fine, once the ticket exists I can take it from there.